### ISSUE TITLE
Unable to calculate test accuracy after every epoch

### ISSUE THREAD

**Issue by AshutoshFreak** [AshutoshFreak/SplitLearningResearchV2#1]
At the time of writing, opacus doesn't support testing after every few epochs. You have to fully train your model before you can do any testing. You cannot continue to train the model after testing.

**Comment by AshutoshFreak** [comment]
This is not due to opacus. I forgot to use torch.no_grad() for testing. Closing the issue now
